misc/TestIAsyncEnumerable/Program.cs

```diff
@@ -14,6 +14,7 @@ class Program
 
         static async Task Main(string[] args)
         {
+            // hi dad! this is a comment
```

Comment:
Hi, Back, can you make data Upper case? 😄

Reply:
no 😠